sdk/publicarchive/helpers.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package publicarchive
+
+import (
+	"fmt"
+	"os"
+)
+
+// IsEmpty returns true if the Downloader is empty and not initialized
+func (d Downloader) IsEmpty() bool {
+	return d.getter.client == nil
+}
+
+func (d Downloader) GetDownloadedFilePath() (string, error) {
+	if d.GetBytesComplete() != d.GetDownloadSize() {
```

Comment:
I should add a IsDownloaded function, and make this one not to file for that condition, 
probably calling it GetFilePath